src/App.global.css

```diff
@@ -2,6 +2,19 @@
 @tailwind components;
 @tailwind utilities;
 
+@import '../node_modules/simplebar-react/dist/simplebar.min.css';
+
+.simplebar-scrollbar:before {
+  background: #808083; /* specify scrollbar color */
```

Comment:
this is where we can define scrollbar color. I went with neutral gray which should work in both dark and light modes.